source/guide_postgresql.rst

```diff
@@ -272,28 +296,283 @@ Step 2 - New Database
  * ``--template``: PostgreSQL supports standard templates to create the database structure.
  * ``database name``: And as last option the name of the database. In this example 'synapse'.
 
-.. warning:: Please replace ``<username>`` with your user name, created in step 1!
+.. warning:: Please replace ``<user>`` with your user name, created in step 1!
 
 .. code-block:: console
  :emphasize-lines: 1
 
- [isabell@stardust ~]$ createdb --encoding=UTF8 --owner=<username> --template=template0 synapse
+ [isabell@stardust ~]$ createdb --encoding=UTF8 --owner=<user> --template=template0 synapse
  [isabell@stardust ~]$
 
 
 Best Practices
 ==============
 
-To configure your project with the PostgreSQL details, you should have the database name, user name and password, localhost as server address and your port number.
+To configure your project with the PostgreSQL details, you should have the database name, user and password, localhost as server address and your port number.
+
+
+Updates
+=======
+
+The update process has some dependencies. Especially the free available space of your Uberspace, because the update process will take temporary ca. the same capacity of your existing data.
```

Comment:
Here the new text: "The update process has one dependency, the free available space of your affected Uberspace. Because the update process will take temporary ca. the same capacity of your existing PostgreSQL instance data for a local backup. After the data migration the backup will be deleted and you will have the same capacity situation as before."